### PR TITLE
formats all files using rustfmt

### PR DESCRIPTION
**Pull request by cssivision**

### CODE REVIEW

**Comment by quininer** [comment]
Thanks, would you like to add a fmt ci? like https://github.com/tokio-rs/tokio/blob/master/.github/workflows/ci.yml#L174

**Comment by cssivision** [comment]
> Thanks, would you like to add a fmt ci? like https://github.com/tokio-rs/tokio/blob/master/.github/workflows/ci.yml#L174

done

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -30,3 +30,20 @@ jobs:
...
+      - uses: actions/checkout@v2
+      - name: Install Rust
+        run: rustup update stable
+      - name: Install rustfmt
+        run: rustup component add rustfmt
+
+      - name: "rustfmt --check"
+        run: |
+          if ! rustfmt --check --edition 2018 $(find . -name '*.rs' -print); then
+            printf "Please run \`rustfmt --edition 2018 \$(find . -name '*.rs' -print)\` to fix rustfmt errors.\nSee CONTRIBUTING.md for more details.\n" >&2
```
Comment (quininer) [inline .github/workflows/rust.yml:47]:
```suggestion
            printf "Please run \`rustfmt --edition 2018 \$(find . -name '*.rs' -print)\` to fix rustfmt errors.\n" >&2
```

**Review by quininer — CHANGES_REQUESTED** [review]

**Review by quininer — APPROVED** [review]

**Comment by quininer** [comment]
Thank you!

**Pull request merged**